handlers/GameHandler.java

```diff
@@ -0,0 +1,68 @@
+package com.kbh.snl.handlers;
+
+import com.kbh.snl.models.Game;
+import com.kbh.snl.models.IntermediateObject;
+import com.kbh.snl.models.Ladder;
+import com.kbh.snl.models.Player;
+import com.kbh.snl.models.Snake;
+import java.util.Iterator;
+import java.util.Random;
+
+public class GameHandler {
+
+  private final Game game;
+  private final Random randomNumberGenerator;
+
+  public GameHandler(final Game game) {
+    this.game = game;
+    this.randomNumberGenerator = new Random();
```

Comment:
This is at a low level. Wrap it with what you want to do with it. As in, have a dice instead.